src/monitor.cc

```diff
@@ -485,16 +553,27 @@ bool NodeMonitor::getBooleanFunction(const char* funcName) {
     return false;
 }
 
+NodeMonitor& NodeMonitor::getInstance() {
+    assert( 0 != instance_ );
+    return *instance_;
+}
+
+/**
+ * Sends udp JSON datagram approximately once per REPORT_INTERVAL_MS
+ *
+ * Executed in the monitr pthread, *not* from any pthread executing v8
+ * Therefore, we can't directly access any Javascript functions/vars
+ **/
 bool NodeMonitor::sendReport() {
     static pid_t pid = getpid();
     static double minOverHead = 0;
 
-    // See how many reports has been processed since last call to this function
-    unsigned int diff_count = instance_->loop_count_ - instance_->last_loop_count_ + 1;
-    instance_->last_loop_count_ = instance_->loop_count_;
+    // See how many reports have been processed since last call to this function
+    unsigned int diff_count = loop_count_ - last_loop_count_ + 1;
+    last_loop_count_ = loop_count_;
 
     // The different between current and previous time (in miliseconds)
```

Comment:
I'm nearly certain that the XeY format means X * 10^Y, so 1.0e6 means "1.0 * 10^6" or 1000000.

... which is why those times look like micro-seconds (millionths of a second) instead of milliseconds (thousandths of a second) to me.